dspace/solr/authority/conf/schema.xml

```diff
@@ -301,7 +301,10 @@
      <field name="institution" type="string" multiValued="false" indexed="true" stored="true" required="false"/>
      <field name="value" type="text" multiValued="false" indexed="true" stored="true" required="false"/>
 
-     <field name="orcid_id" type="string" multiValued="false" indexed="true" stored="true" required="false"/>
+   <field name="value_keyword" type="string" multiValued="true" indexed="true" stored="true" required="false"/>
```

Comment:
Just calling a field "_keyword" doesn't imply the "keyword" functionality out of the box. You need to port this configuration https://github.com/atmire/RIOXX/blob/master/dspace/solr/search/conf/schema.xml#L569 including the keywordFilter to the authority solr schema.